src/foam/u2/detail/SectionedDetailPropertyView.js

```diff
@@ -89,8 +88,6 @@ foam.CLASS({
     ^error .foam-u2-DateView,
     ^error .foam-u2-view-date-DateTimePicker .date-display-box
     {
-      color: /*%DESTRUCTIVE3%*/ #d9170e !important;
-      background-color: /*%DESTRUCTIVE5%*/ #fbedec !important;
       border-color: /*%DESTRUCTIVE3%*/ #d9170e !important;
```

Comment:
Should border-color still be red as well?

Reply:
I'll leave the border color. I think it's valuable for scrolling through a long detail view and being able to catch them. If somebody disagrees it's an easy enough change to make.